### ISSUE TITLE
add AS2WASM to dist 

### ISSUE THREAD

**Issue by blackjyn** [apache/royale-asjs#108]
Please includes AS2WASM build to Royale distribution 
Flash and Flex folks are now looking for SWF->WASM 

Thx

**Comment by ROBERT-MCDOWELL** [comment]
thanks  blackjyn to point it out...

**Comment by bigosmallm** [comment]
@erikdebruin Do you want to take this up?

**Comment by kennylerma** [comment]
@erikdebruin  Is this still being worked on?  Is there still hope for WASM?

**Comment by blackjyn** [comment]
Other people is trying to do so, however the API impl is incomplete 
WASM target is really great for Royale
https://github.com/JasonHuang3D/AJC-Flash-WebAssembly-Examples

**Comment by kennylerma** [comment]
Yeah, I've actually talked with Jason Huang about the project.  It's using a modified version of the Falcon compiler and it's looking good so far, but still lots to do.  I haven't seen an update in while though.

I definitely want to see WASM with Royale.  This is where everything is headed, so I hope Royale can get ahead on this.
